po/de.po

```diff
@@ -18610,7 +18610,7 @@ msgstr "Ausführliche _Fehlermeldungen"
 #: gnucash/import-export/aqb/dialog-ab-pref.glade:76
 #: gnucash/import-export/aqb/gschemas/org.gnucash.GnuCash.dialogs.import.hbci.gschema.xml.in:31
 msgid "Enables verbose debug messages for HBCI/AqBanking Online Banking."
-msgstr "Ausführliche Fehlermeldungen für HBCI Online Banking aktivieren."
+msgstr "Ausführliche Fehlermeldungen für das FinTS Online Banking aktivieren."
```

Comment:
https://wiki.gnucash.org/wiki/AqBanking#Debugging: Since Gnucash 2.3.x this is related to AQBANKING_LOGLEVEL.
also für ganz AqBanking

Reply:
"FinTS/" weg.